tests/pages/jobs/JobActions-cy.js

```diff
@@ -13,9 +13,7 @@ describe("Job Actions", function() {
     });
 
     it("opens the correct jobs edit modal", function() {
-      cy
-        .get('.modal .form-panel input[name="id"]')
-        .should("to.have.value", "foo");
+      cy.get(".modal .form-panel form input").should("to.have.value", "foo");
```

Comment:
@nLight @orlandohohmeier This is needed because after the bump I notice this behavior with the input fields. On the left is the before on the right is the after. It looks like it sanitizes the properties which is good in a way, is this expected behavior?

![image](https://user-images.githubusercontent.com/180432/31477452-f7783d7c-aec0-11e7-8bb6-af53d8cd5a21.png)

Reply:
Hey @bstavroulakis! It is an artifact from an another update. So the filtering is indeed intentional, though we still want to keep the name attribute. I'm going to create a fix PR for the components. Sorry for any trouble.

Reply:
https://github.com/mesosphere/reactjs-components/pull/416

Reply:
We gonna have another bump